### PR TITLE
Generate export segment and export sub segment data within Elasticsearch

### PR DESCRIPTION
**Pull request by vfarah-if**
Extended Company to facilitate export segments and sub-segments and exposing the same information within ES.

### Checklist

* [x] If this is a releasable change, has a news fragment been added?

  <details>
  <summary>Explanation</summary>
  
  A news fragment is required for any releasable change (i.e. code that runs in or affects production) so that a corresponding changelog entry is added when releasing.
  
  Check [changelog/README.md](https://github.com/uktrade/data-hub-api/blob/master/changelog/README.md) for instructions.
  
  </details>
  
* [x] Has this branch been rebased on top of the current `develop` branch?

  <details>
  <summary>Explanation</summary>
  
  The branch should not be stale or have conflicts at the time reviews are requested.
  
  </details>

* [x] Is the CircleCI build passing?

### General points

<details>
<summary><strong>Other things to check</strong></summary><p></p>

* Make sure `fixtures/test_data.yaml` is maintained when updating models
* Consider the admin site when making changes to models
* Use select-/prefetch-related field lists in views and search apps, and update them when fields are added
* Make sure the README is updated e.g. when adding new environment variables

</details>

See [docs/CONTRIBUTING.md](https://github.com/uktrade/data-hub-api/blob/develop/docs/CONTRIBUTING.md) for more guidelines.

### CODE REVIEW

File: datahub/core/constants.py [hunk 1]
```diff
@@ -362,3 +362,38 @@ class InvestmentStrategicDriver(Enum):
     access_to_market = Constant(
         'Access to market', '382aa6d1-a362-4166-a09d-f579d9f3be75',
     )
+
+
+class ExportSegment(Enum):
+    """ExportSegment type constants."""
+
+    hep = Constant('hep', 'High export potential')
+    non_fdi = Constant('non-hep', 'Not high export potential')
```
Comment (chloehkwong1) [inline datahub/core/constants.py:371]: Why is this variable called `non_fdi` and not `non_hep`?

Comment (vfarah-if) [reply datahub/core/constants.py]: Fixed

File: datahub/company/migrations/0109_add_export_segment_and_subsegment.py [hunk 2]
```diff
@@ -18,6 +18,6 @@ class Migration(migrations.Migration):
         migrations.AddField(
```
Comment (PippoRaimondi) [inline datahub/company/migrations/0109_add_export_segment_and_subsegment.py:18]: shouldn't this also add `export_segment` field?

Comment (vfarah-if) [reply datahub/company/migrations/0109_add_export_segment_and_subsegment.py:18]: This is a branch of a branch and i think that changes was done in the CDS-94 branch which is pending. Let me know if that makes sense?

**Review by PippoRaimondi — APPROVED** [review]

File: changelog/cds-95-export-segment.feature.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Extended Elastic data to expose export segment and sub segment data.
```
Comment (ambrozic) [inline changelog/cds-95-export-segment.feature.md:1]: Can you change `Elastic` to `Elasticsearch`?

**Review by ambrozic — APPROVED** [review]

**Review by elcct — APPROVED** [review]

File: changelog/cds-95-export-segment.feature.md (hunk 3 above)
Comment (chloehkwong1) [reply changelog/cds-95-export-segment.feature.md]: @ambrozic done :)

**Pull request merged**